spec/order_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'order'
+
+describe Order do
+  let(:menu_double) { double :Menu, list: { :shawarma => "£5", :chips => "£3", :salad => "£2", :pepsi => "£1" } }
+  
+  it 'returns the menu on #menu' do
+    order = Order.new(menu_double)
+    expect(order.menu).to include(
+      :shawarma => "£5",
+      :chips => "£3",
+      :salad => "£2",
+      :pepsi => "£1"
+    )
+  end
+
+  it 'allows the user to #add to their order' do
```

Comment:
Just adding to comment on order.rb:13. You're test context says '#add' but is testing `.update`. Just for clarity I'd look at making them the same as each other. (Just a small thought)